lesson_02/dags/dag_load_local_to_gcs.py

```diff
@@ -0,0 +1,25 @@
+from datetime import datetime
+import os
+from airflow import DAG
+from airflow.providers.google.cloud.transfers.local_to_gcs import LocalFilesystemToGCSOperator
+
+os.environ["GOOGLE_APPLICATION_CREDENTIALS"] = "../../de2022-stanislav-rezen-4dd944e14701.json"
+
+with DAG(
+    dag_id='dag_load_local_to_gcs',
+    start_date=datetime(2022, 8, 1),
+    end_date=datetime(2022, 8, 3),
+    schedule_interval="0 1 * * *",
+    catchup=True,
+    max_active_runs=1
```

Comment:
This limitation should not be used here because you feel free to run many dag runs at the same time
(Of course, we have a physical limitation when we run Airflow locally for development and testing)